### ISSUE TITLE
Wrong hc-transform causing bad lat long on a map of Michigan state

### ISSUE THREAD

**Issue by KacperMadej** [highcharts/map-collection-dist#40]
The Highmaps lat long related definitions are incorrect for the map of Michigan state. 
Demo: http://jsfiddle.net/pabvce4f/

**Original report:**
 https://github.com/highcharts/highcharts/issues/4084#issuecomment-518862739

**As a workaround:**
> yoink the hc-transform from the congressional map, it works: http://jsfiddle.net/s5bLjcgw/

**Comment by zxlin** [comment]
@KacperMadej any updates on this or how can I help resolve this issue?

**Comment by KacperMadej** [comment]
A workaround is provided. Currently, we do not have ETA for the fix.

**Comment by victorvaldez5** [comment]
Do you guys have an ETA for this fix yet? It's been 6 years

**Comment by KacperMadej** [comment]
Hi @victorvaldez5 

We didn't have many requests for this so the assumption was that the workaround is enough or this is not a widely popular problem.

It works fine when using the currently recommended map file format - TopoJSON: https://jsfiddle.net/BlackLabel/wf5cp1no/
GeoJSON is a legacy format, so the priority for this bug is very low.

I doubt this bug will be fixed in any other way than as a side effect of something else - like changing map source to Open Street Map due to some other changes / requirements.

tl;dr No ETA.
